odp_sdk/resource_client.py

```diff
@@ -14,7 +14,7 @@ class OdpResourceClient(BaseModel):
     """Client for interacting with ODP resources."""
 
     http_client: OdpHttpClient
-    resource_endpoint: str
+    resource_endpoint: str = "/catalog"
```

Comment:
This attribute should not have a default value, instead it should be set in the `OdpClient`

Reply:
Fixed